project/Dependencies.scala

```diff
@@ -17,16 +17,16 @@ object Dependencies {
   val munitCatsEffectV = "1.0.7"
   val pact4sV = "0.9.0"
 
-  private val workbenchLibsHash = "01a11c3"
-  val serviceTestV = s"2.1-$workbenchLibsHash"
-  val workbenchModelV = s"0.17-$workbenchLibsHash"
+  private val workbenchLibsHash = "d764a9b"
+  val serviceTestV = s"3.1-$workbenchLibsHash"
```

Comment:
This is a major version bump - is there a reason Leo's version of serviceTest was sitting at 2.1?

Reply:
Resolved